security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -178,7 +178,15 @@ func NewSecretController(ca ca.CertificateAuthority, certTTL time.Duration, trus
 // Run starts the SecretController until a value is sent to stopCh.
 func (sc *SecretController) Run(stopCh chan struct{}) {
 	go sc.scrtController.Run(stopCh)
-	go sc.saController.Run(stopCh)
+	for {
```

Comment:
replace with `cache.WaitForCacheSync()`

Reply:
Please fix as suggested by @hzxuzhonghu

Reply:
@hzxuzhonghu @myidpt Thanks for your suggestion. I am working on